src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -622,14 +623,19 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
       if (!state?.withdraw.cryptoAmount) return
 
       if (dayjs().isAfter(dayjs.unix(Number(expiry)))) {
-        toast({
-          position: 'top-right',
-          description: translate('trade.errors.quoteExpired'),
-          title: translate('common.transactionFailed'),
-          status: 'error',
+        console.debug({
+          now: dayjs().format('YYYY-MM-DDTHH:mm:ss'),
+          expiry: dayjs.unix(Number(expiry)).format('YYYY-MM-DDTHH:mm:ss'),
```

Comment:
So we can get insights on what's happening here

Reply:
Actually we may want to add some Mixpanel tracking here

Reply:
169925f88ad22dee052a4728acf002491c1e1cc2

Reply:
can you also print the unix timestamps `dayjs().unix()` and `expiry` as well to validate without the tz conversions?

Reply:
Absolutely - 946c981ec7